autogpt_platform/backend/pyproject.toml

```diff
@@ -90,3 +90,6 @@ ignore_patterns = []
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
+
+[tool.ruff]
+target-version = "py310"
```

Comment:
without this, `ruff` complains about using `anext(aiter(..))`, since those functions were only added in Python 3.10